### ISSUE TITLE
Segementation fault on WSL

### ISSUE THREAD

**Issue by Yakagi17** [automl/auto-sklearn#1047]
## Describe the bug ##
I tried to run a simple example of auto-classification from Auto-Sklearn documentation API but got error.

code :
import sklearn.datasets
import sklearn.metrics

import autosklearn.classification
 if __name__ == "__main__":

    X, y = sklearn.datasets.load_breast_cancer(return_X_y=True)
    X_train, X_test, y_train, y_test = \
        sklearn.model_selection.train_test_split(X, y, random_state=1)

    automl = autosklearn.classification.AutoSklearnClassifier()
    automl.fit(X_train, y_train, dataset_name='breast_cancer')

## Expected behavior ##
I tried to fit a simple auto-classification model then get model accuracy but that happend

## Actual behavior, stacktrace or logfile ##
/home/zap/.local/lib/python3.8/site-packages/pyparsing.py:3190: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
/home/zap/.local/lib/python3.8/site-packages/pyparsing.py:3190: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
/home/zap/.local/lib/python3.8/site-packages/pyparsing.py:3190: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
/home/zap/.local/lib/python3.8/site-packages/pyparsing.py:3190: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
Segmentation fault

## Environment and installation: ##
* Ubuntu 20.04.1 from WSL(Windows Sub Linux)
* IInstall using pip3 install
* Python 3.8.5
* Auto-sklearn version 0.12.0
* swig 3.0.12

other package installed
altair                 4.1.0               
appdirs                1.4.4               
argon2-cffi            20.1.0              
astor                  0.8.1               
astroid                2.4.2               
async-generator        1.10                
attrs                  19.3.0              
auto-sklearn           0.12.0              
Automat                0.8.0               
backcall               0.2.0               
base58                 2.0.1               
bleach                 3.2.1               
blinker                1.4                 
cachetools             4.2.0               
certifi                2019.11.28          
cffi                   1.14.4              
chardet                3.0.4               
Click                  7.0                 
cloud-init             20.2                
cloudpickle            1.6.0               
colorama               0.4.3               
command-not-found      0.3                 
configobj              5.0.6               
ConfigSpace            0.4.16              
constantly             15.1.0              
cryptography           2.8                 
Cython                 0.29.21             
dask                   2020.12.0           
dbus-python            1.2.16              
decorator              4.4.2               
defusedxml             0.6.0               
distlib                0.3.1               
distributed            2020.12.0           
distro                 1.4.0               
distro-info            0.23ubuntu1         
docutils               0.16                
entrypoints            0.3                 
filelock               3.0.12              
gitdb                  4.0.5               
GitPython              3.1.11              
HeapDict               1.0.1               
httplib2               0.14.0              
hyperlink              19.0.0              
idna                   2.8                 
importlib-metadata     1.5.0               
incremental            16.10.1             
ipykernel              5.4.2               
ipython                7.19.0              
ipython-genutils       0.2.0               
ipywidgets             7.6.2               
isort                  5.6.4               
jedi                   0.18.0              
Jinja2                 2.10.1              
joblib                 1.0.0               
jsonpatch              1.22                
jsonpointer            2.0                 
jsonschema             3.2.0               
jupyter-client         6.1.7               
jupyter-core           4.7.0               
jupyterlab-pygments    0.1.2               
jupyterlab-widgets     1.0.0               
keyring                18.0.1              
language-selector      0.1                 
launchpadlib           1.10.13             
lazr.restfulclient     0.14.2              
lazr.uri               1.0.3               
lazy-import            0.2.2               
lazy-object-proxy      1.4.3               
liac-arff              2.5.0               
MarkupSafe             1.1.0               
mccabe                 0.6.1               
mistune                0.8.4               
more-itertools         4.2.0               
msgpack                1.0.2               
nbclient               0.5.1               
nbconvert              6.0.7               
nbformat               5.0.8               
nest-asyncio           1.4.3               
netifaces              0.10.4              
notebook               6.1.6               
numpy                  1.19.4              
oauthlib               3.1.0               
packaging              20.8                
pandas                 1.2.0               
pandocfilters          1.4.3               
parso                  0.8.1               
pexpect                4.8.0               
pickleshare            0.7.5               
Pillow                 8.0.1               
pip                    20.0.2              
pipenv                 2020.11.15          
prettytable            2.0.0               
prometheus-client      0.9.0               
prompt-toolkit         3.0.8               
protobuf               3.14.0              
psutil                 5.8.0               
ptyprocess             0.7.0               
pyarrow                2.0.0               
pyasn1                 0.4.2               
pyasn1-modules         0.2.1               
pycparser              2.20                
pydeck                 0.5.0               
pyecharts              1.9.0               
Pygments               2.7.3               
PyGObject              3.36.0              
PyHamcrest             1.9.0               
PyJWT                  1.7.1               
pylint                 2.6.0               
pymacaroons            0.13.0              
PyNaCl                 1.3.0               
pynisher               0.6.3               
pyOpenSSL              19.0.0              
pyparsing              2.4.7               
pyrfr                  0.8.0               
pyrsistent             0.15.5              
pyserial               3.4                 
python-apt             2.0.0+ubuntu0.20.4.1
python-dateutil        2.8.1               
python-debian          0.1.36ubuntu1       
pytz                   2020.5              
PyYAML                 5.3.1               
pyzmq                  20.0.0              
requests               2.22.0              
requests-unixsocket    0.2.0               
scikit-learn           0.23.2              
scipy                  1.5.4               
SecretStorage          2.3.1               
Send2Trash             1.5.0               
service-identity       18.1.0              
setuptools             45.2.0              
simplejson             3.16.0              
six                    1.14.0              
smac                   0.13.1              
smmap                  3.0.4               
sortedcontainers       2.3.0               
ssh-import-id          5.10                
streamlit              0.73.1              
streamlit-echarts      0.1.0               
systemd-python         234                 
tblib                  1.7.0               
terminado              0.9.1               
testpath               0.4.4               
threadpoolctl          2.1.0               
toml                   0.10.2              
toolz                  0.11.1              
tornado                6.1                 
traitlets              5.0.5               
Twisted                18.9.0              
tzlocal                2.1                 
ubuntu-advantage-tools 20.3                
ufw                    0.36                
unattended-upgrades    0.1                 
urllib3                1.25.8              
validators             0.18.2              
virtualenv             20.2.2              
virtualenv-clone       0.5.4               
wadllib                1.3.3               
watchdog               1.0.2               
wcwidth                0.2.5               
webencodings           0.5.1               
wheel                  0.34.2              
widgetsnbextension     3.5.1               
wrapt                  1.12.1              
zict                   2.0.0               
zipp                   1.0.0               
zope.interface         4.7.1

**Comment by Yakagi17** [comment]
fixed with installing Ubuntu 18.04
